apps/fluent-tester/src/E2E/Shadow/specs/Shadow.spec.macos.ts

```diff
@@ -0,0 +1,21 @@
+import NavigateAppPage from '../../common/NavigateAppPage';
+import ShadowPageObject from '../pages/ShadowPageObject';
+import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT } from '../../common/consts';
+
+// Before testing begins, allow up to 60 seconds for app to open
+describe('Shadow Testing Initialization', function () {
+  it('Wait for app load', async () => {
+    await NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
+    await expect(await NavigateAppPage.isPageLoaded()).toBeTruthy(NavigateAppPage.ERRORMESSAGE_APPLOAD);
+  });
+
+  it('Click and navigate to Shadow test page', async () => {
+    await ShadowPageObject.waitForButtonDisplayed();
```

Comment:
Do we need this check? We're not performing an UI beforehand, so is it necessary>

Reply:
I think this is a leftover from other mac specs. Maybe a separate PR to remove these?

Reply:
Yes, I'd recommend making a new PR and cleaning this up